src/formatISO9075/index.js

```diff
@@ -0,0 +1,111 @@
+import toDate from '../toDate/index.js'
+import isValid from '../isValid/index.js'
+import addLeadingZeros from '../_lib/addLeadingZeros/index.js'
+
+/**
+ * @name formatISO9075
+ * @category Common Helpers
+ * @summary Format the date according to the ISO 9075 standard (https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_get-format).
+ *
+ * @description
+ * Return the formatted date string in ISO 9075 format. Options may be passed to control the parts and notations of the date.
+ *
+ * @param {Date|Number} date - the original date
+ * @param {Object} [options] - an object with options.
+ * @param {'extended'|'basic'} [options.format='extended'] - if 'basic', doesn't show delimeters between date and time values.
```

Comment:
```suggestion
 * @param {'extended'|'basic'} [options.format='extended'] - if 'basic', hide delimiters between date and time values.
```
same thing like the previous one in `formatISO`